proto/hub-genesis/genesis.proto

```diff
@@ -3,13 +3,13 @@ package rollapp.hub_genesis;
 
 import "gogoproto/gogo.proto";
 import "hub-genesis/params.proto";
-import "hub-genesis/hub.proto";
+import "hub-genesis/locked.proto";
 
 option go_package = "github.com/dymensionxyz/dymension-rdk/x/hub-genesis/types";
 
 // GenesisState defines the hub-genesis module's genesis state.
 message GenesisState {
   // params defines all the parameters of the module.
   Params params = 1 [ (gogoproto.nullable) = false ];
-  Hub hub = 2 [ (gogoproto.nullable) = false ];
+  Locked locked = 2 [(gogoproto.nullable) = false];
```

Comment:
`locked` is fine, but maybe also name it that on the hub, so it's consistent?

Reply:
I think `locked` is confusing as an indicator if the genesis event already ran or not as it's implicit. 
I think it should be more explicit (especially if we will do in the future more operations in the genesis event besides locking). 
I believe it should be similar to the hub, i.e

```
  // is_genesis_event is a boolean that indicates if the genesis event has occured
  bool is_genesis_event = 2;
```